backend/app/controllers/spree/admin/refunds_controller.rb

```diff
@@ -8,6 +8,22 @@ class RefundsController < ResourceController
 
       rescue_from Spree::Core::GatewayError, with: :spree_core_gateway_error
 
+      def create
+        @refund.attributes = refund_params
+        if @refund.save
+          @refund.perform!
+          flash[:success] = flash_message_for(@refund, :successfully_created)
+          respond_with(@refund) do |format|
+            format.html { redirect_to location_after_save }
+          end
+        else
+          flash[:error] = @refund.errors.full_messages.join(", ")
```

Comment:
This needs to be `flash.now[:error]` right? (Though if we use the `RefundController#create` this goes away anyway.